meta/main.yml

```diff
@@ -2,9 +2,13 @@
 dependencies: []
 
 galaxy_info:
-  author: defunctzombie
+  author: yaman
   description: "bootstrap coreos hosts to run ansible"
   license: "MIT"
   min_ansible_version: 1.4
+  platforms:
```

Comment:
What is the connection of this change to the proposed theme of your pull request, namely, `--no-check-certificate`? 

I think this belongs to a different pull request.